wire-ios-data-model/Source/Model/Conversation/ZMConversation+Participants.swift

```diff
@@ -258,14 +258,14 @@ extension ZMConversation {
                 do {
                     try await mlsService.addMembersToConversation(with: mlsUsers, for: mlsGroupID)
 
-                    context.perform {
+                    await context.perform {
```

Comment:
why?

Reply:
`perform` is async method for iOS 15 and therefore `await` is required

```
@available(macOS 12.0, iOS 15.0, tvOS 15.0, watchOS 8.0, *)
extension NSManagedObjectContext {

    public func performAndWait<T>(_ block: () throws -> T) rethrows -> T

    public func perform<T>(schedule: NSManagedObjectContext.ScheduledTaskType = .immediate, _ block: @escaping () throws -> T) async rethrows -> T
}
```